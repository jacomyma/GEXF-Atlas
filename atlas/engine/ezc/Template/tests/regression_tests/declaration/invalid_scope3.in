{foreach array(1) as $a}
    {foreach array(1) as $q}
        {var $b}
    {/foreach}
{/foreach}
