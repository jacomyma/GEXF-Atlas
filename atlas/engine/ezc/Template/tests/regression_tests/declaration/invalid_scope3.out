mock:3:16: The variable <$b> is cannot be declared in a subblock: while, foreach, if, etc

        {var $b}
               ^
